bambou/session.go

```diff
@@ -361,7 +398,7 @@ func (s *Session) FetchChildren(parent Identifiable, identity Identity, dest int
 
 	request, err := http.NewRequest("GET", url, nil)
 	if err != nil {
-		return NewError(http.StatusBadRequest, err.Error())
+		return NewBambouError("", err.Error())
```

Comment:
Why are we not returning the bad request code as title? Not a big fan of empty titles